File: dataset/human/python/file_5566.py
void main(){
  import std.stdio, std.string, std.conv, std.algorithm;
  import std.math;
  
  auto s=readln.chomp.to!(char[]);

  char[] t;
  foreach(c; s)if(c!='x') t~=c;
  bool ok=true;
  foreach(i; 0..(t.length/2)) ok&=(t[i]==t[$-i-1]);
  if(!ok){writeln(-1); return;}

  int cnt=0;
  int[] y;
  foreach(c; s){
    if(c=='x') cnt++;
    else y~=cnt, cnt=0;
  }
  y~=cnt;
  int sum=0;
  foreach(i; 0..(y.length/2)) sum+=(y[i]-y[$-i-1]).abs;
  writeln(sum);
}

void rd(T...)(ref T x){
  import std.stdio, std.string, std.conv;
  auto l=readln.split;
  foreach(i, ref e; x){
    e=l[i].to!(typeof(e));
  }
}

File: dataset/human/c/file_5566.c
#include <stdio.h>
#include <math.h>
#define MAX 104730

int num[MAX];

int main(void){
	int i, j, in, cnt, sum;
	for(i = 2; i < sqrt(MAX); i++){
		for(j = i; i * j < MAX; j++)	num[i*j] = 1;
	}

	while(1){
		scanf("%d", &in);
		if(in == 0)	break;
		cnt = 0; sum = 0;
		for(i = 2; i < MAX; i++){
			if(cnt == in)	break;
			if(num[i] == 0){
				sum += i;
				cnt++;
			}
		}
		printf("%d\n", sum);
	}

  return 0;
}
